src/auth/digest/eDirectory/ldap_backend.cc

```diff
@@ -204,7 +204,9 @@ getpassword(char *login, char *realm)
     char *password = NULL;
     int retry = 0;
     char filter[8192];
-    char searchbase[8192];
+    *filter='\0';
+    char searchbase[8192]
+    *searchbase='\0';
```

Comment:
Just formatting (or consistency) improvements if you get a chance:

```suggestion
    *filter = '\0';
    char searchbase[8192]
    *searchbase = '\0';
```